xdsl/ir.py

```diff
@@ -21,6 +22,14 @@ class MLContext:
     _registeredOps: dict[str, type[Operation]] = field(default_factory=dict)
     _registeredAttrs: dict[str, type[Attribute]] = field(default_factory=dict)
 
+    def register_dialect(self, dialect: types.ModuleType):
```

Comment:
I did not figure out what the proper `typing` import is. If anyone knows, please point out!